source/Files.cpp

```diff
@@ -231,6 +231,13 @@ const string &Files::Tests()
 
 
 
+const string Files::Plugins()
```

Comment:
It should be returning a reference (in other places too).

Reply:
right